product_abc_classification/models/abc_classification_profile.py

```diff
@@ -71,7 +71,26 @@ def _check_levels(self):
                 )
 
     def _compute_abc_classification(self):
-        raise NotImplementedError()
+        ProductClassification = self.env["abc.classification.product.level"]
+        for profile in self:
```

Comment:
```suggestion
        for profile in self.filtered(lambda p: p.profile_type == "manual"):
```
?

Reply:
This is handled in the modules extending the base one and adding new profiles. See an example here: https://github.com/OCA/product-attribute/blob/93e4b9cefe0a62564de6c7b8302f3442125e7337/product_abc_classification_sale_stock/models/abc_classification_profile.py#L236